sklearn/manifold/t_sne.py

```diff
@@ -767,10 +771,10 @@ def _fit(self, X, skip_num_points=0):
             pca = PCA(n_components=self.n_components, svd_solver='randomized',
                       random_state=random_state)
             X_embedded = pca.fit_transform(X)
-        elif isinstance(self.init, np.ndarray):
-            X_embedded = self.init
         elif self.init == 'random':
```

Comment:
Looks like there's one more place that would throw a warning for the same reason in the event that metric='precomputed'. I'll fix this one as well using a type check first. Same question as above in terms of whether it's a good idea to write a test to check that a FutureError isn't thrown?